Protacon.RxMq.AzureServiceBus/Topic/AzureTopicSubscriber.cs

```diff
@@ -84,21 +110,55 @@ public AzureTopicSubscriber(IOptions<AzureBusTopicSettings> settings, AzureBusTo
             _settings = settings.Value;
             _queueManagement = queueManagement;
             _logging = logging;
-        }
 
+            _source = new CancellationTokenSource();
+            Task.Factory.StartNew(() =>
```

Comment:
Add "LongRunning" flag to this task, otherwise it will not tell threadpool to fetch new thread for other tasks. It occurs after while via some magical algorithm but anyway.